backend/data/processors/manualFixes/FINSFixes.py

```diff
@@ -0,0 +1,156 @@
+"""
+https://github.com/csesoc/Circles/wiki/Manual-Fixes-to-Course-Prerequisites
+
+Copy this into a new file for the relevant faculty's fixes: 
+e.g. COMPFixes.py, ACCTFixes.py, PSYCFixes.py
+
+Apply manual FINS fixes to processed conditions in conditionsProcessed.json so 
+that they can be fed into algorithms.
+
+If you make a mistake and need to regenerate conditionsProcessed.json, then you
+can run:
+    python3 -m data.processors.conditionsPreprocessing
+
+To then run this file:
+    python3 -m data.processors.manualFixes.FINSFixes
+"""
+
+from data.utility import dataHelpers
+
+# Reads conditionsProcessed dictionary into 'CONDITIONS'
+CONDITIONS = dataHelpers.read_data("data/finalData/conditionsProcessed.json")
+PROCESSED = "processed"
+
+# Reads coursesProcessed dictionary into 'COURSES' (for updating exclusions)
+COURSES = dataHelpers.read_data("data/finalData/coursesProcessed.json")
+
+
+def fix_conditions():
+    """ Functions to apply manual fixes """
+
+    CONDITIONS["FINS2101"][PROCESSED] = FINS_2101()
+    CONDITIONS["FINS2622"][PROCESSED] = FINS_2622()
+    CONDITIONS["FINS3202"][PROCESSED] = FINS_3202_3303()
+    CONDITIONS["FINS3303"][PROCESSED] = FINS_3202_3303()
+    CONDITIONS["FINS3626"][PROCESSED] = FINS_3626()
+    CONDITIONS["FINS3630"][PROCESSED] = FINS_3630()
+    CONDITIONS["FINS3645"][PROCESSED] = FINS_3645_6()
+    CONDITIONS["FINS3646"][PROCESSED] = FINS_3645_6()
+    CONDITIONS["FINS4774"][PROCESSED] = FINS_4774()
+    CONDITIONS["FINS4775"][PROCESSED] = FINS_4775()
+    CONDITIONS["FINS4776"][PROCESSED] = FINS_4776_7_9()
+    CONDITIONS["FINS4777"][PROCESSED] = FINS_4776_7_9()
+    CONDITIONS["FINS4779"][PROCESSED] = FINS_4776_7_9()
+    CONDITIONS["FINS4792"][PROCESSED] = FINS_4792()
+
+    # Updates the files with the modified dictionaries
+    dataHelpers.write_data(
+        CONDITIONS, "data/finalData/conditionsProcessed.json")
+    dataHelpers.write_data(COURSES, "data/finalData/coursesProcessed.json")
+
+
+def FINS_2101():
+    """ 
+    "original": "Enrolled in plan FINSD13554 or in FINSBH3565<br/><br/>",
+
+    "processed": "plan FINSD13554 || in FINSBH3565"
+    """
+
+    # Legacy Handbook
+    return "FINSD13554 || FINSBH3565"
+
+
+def FINS_2622():
+    """
+    "original": "Prerequisite or co-requisite:FINS1612 and FINS2624<br/><br/>",
+
+    "processed": "|| [FINS1612 && FINS2624]"
+    """
+
+    # BOTH prerequisite and corequisite
+    return "(FINS1612 && FINS2624) || [FINS1612 && FINS2624]"
```

Comment:
If co-req, should this just be `[FINS1612 && FINS2624]`; i.e. do we still need the `(FINS1612 && FINS2624)`? I think we should stick with just having `[FINS1612 && FINS2624]` because the regex successfully does this in some instances. Thoughts?

Reply:
I thought the same initially but turns out, co-rerequisite means have to take in the same term (and only in the same term). So if we just had [FINS1612 && FINS2624], then they have to take FINS1612 and FINS2624 in the same term. So if it's just corerequisite, then even if they took FINS1612 in a prior term, it wouldn't count

Reply:
Are you sure about that? The glossary says "A course that must be enrolled in concurrently with another course (**unless it has already been completed**), usually due to a relationship between the courses in the program curriculum." This means that a student can complete the course prior to the target course. I think the [ ] should include all previous terms and the term being enrolled in.

Reply:
gg you're right

Reply:
I've made the changes, squash & merge if it looks good